### PR TITLE
Empty grid

### PR DESCRIPTION
**Pull request by varunREDDY4**

### CODE REVIEW

File: battleship.py [hunk 1]
```diff
@@ -62,7 +90,14 @@ def mousePressed(data, event, board):
...
 def emptyGrid(rows, cols):
-    return
+    grid =[]
+    for i in range(rows):
+        # an empty list is created
+        boardString=[]
+        for j in range(cols):
+            boardString.append(1)
+        grid.append(boardString)
+    return grid
```
Comment (LaxmiNarayanaMurthyVemuri) [inline battleship.py:100]: The code snippet for the empty grid function is ok. Why do we have remaining functionality in this branch?

**Review by LaxmiNarayanaMurthyVemuri — CHANGES_REQUESTED** [review]

File: battleship.py (hunk 1 above)
Comment (varunREDDY4) [reply battleship.py:100]: Sir, as I have written all the functions before and then applied push so this is happening

Comment (LaxmiNarayanaMurthyVemuri) [reply battleship.py:100]: It is okay for a function, but all the pull requests are similar. It would be best if you reworked on it

Comment (varunREDDY4) [reply battleship.py:100]: okay sir will be starting from scratch again one after another pull request

Comment (varunREDDY4) [reply battleship.py:100]: will delete these branches and write again.

**Pull request closed without merging**